clients/python-wrapper/lakefs/branch.py

```diff
@@ -1,12 +1,112 @@
 """
 Module containing lakeFS branch implementation
 """
+from __future__ import annotations
 
-from lakefs.reference import Reference
+import lakefs_sdk
+from lakefs.object_io import WriteableObject
+from lakefs.object_manager import WriteableObjectManager
+from lakefs.reference import Reference, Commit
+from lakefs.exceptions import api_exception_handler, ConflictException, LakeFSException
 
 
 class Branch(Reference):
     """
     Class representing a branch in lakeFS.
     """
-    # TODO: Implement
+
+    def create(self, source_reference_id: str, exist_ok: bool = False) -> Branch:
+        """
+        Create a new branch in lakeFS from this object
```

Comment:
For all docstrings:
Newline between function description and parameters (part of the docs generation sphinx requirement)